PlatformAwareEnergy/README.md

```diff
@@ -0,0 +1,118 @@
+# Intel's Platform-Aware-Scheduling
+
+This documentation copies parts of the documentation from the [main repo](https://github.com/intel/platform-aware-scheduling). 
+
+
+Platform Aware Scheduling (PAS) contains a group of related projects designed to expose platform specific attributes to the Kubernetes scheduler using a modular policy driven approach. The project contains a core library and information for building custom scheduler extensions as well as specific implementations that can be used in a working cluster or leveraged as a reference for creating new Kubernetes scheduler extensions.
+
+Telemetry Aware Scheduling is the initial reference implementation of Platform Aware Scheduling. It can expose any platform-level metric to the Kubernetes Scheduler for policy driven filtering and prioritization of workloads. You can read more about TAS [here](https://github.com/intel/platform-aware-scheduling/tree/master/telemetry-aware-scheduling).
+
+## Telemetry Aware Scheduling
+
+Telemetry Aware Scheduler Extender is contacted by the generic Kubernetes Scheduler every time it needs to make a scheduling decision. The extender checks if there is a telemetry policy associated with the workload. If so, it inspects the strategies associated with the policy and returns opinions on pod placement to the generic scheduler. The scheduler extender has two strategies it acts on - scheduleonmetric and dontschedule. This is implemented and configured as a [Kubernetes Scheduler Extender](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#cluster-level-extended-resources). 
+
+The Scheduler consumes TAS Policies - which is a Custom Resource. The extender parses this policy for deschedule, scheduleonmetric and dontschedule strategies and places them in a cache to make them locally available to all TAS components. It consumes new Telemetry Policies as they are created, removes them when deleted, and updates them as they are changed. The extender also monitors the current state of policies to see if they are violated. For example if it notes that a deschedule policy is violated it labels the node as a violator allowing pods relating to that policy to be descheduled.
+
+### Quick Setup
+
+The [deploy folder](https://github.com/intel/platform-aware-scheduling/tree/master/telemetry-aware-scheduling/deploy) has all of the yaml files necessary to get Telemetry Aware Scheduling running in a Kubernetes cluster. Some additional steps are required to configure the generic scheduler and metrics endpoints. Here are the steps:
```

Comment:
The link `https://github.com/intel/platform-aware-scheduling/tree/master/telemetry-aware-scheduling/deploy` now references a branch, but I expect `master` branch will move on, so you may want to use whatever commit HEAD is at to permalink the document.